File: BAEKJOON/Node/2000/2500/2530.js
const fs = require("fs")
const filepath = process.platform === "linux" ? "/dev/stdin" : "./input.txt"
const input = fs.readFileSync(filepath).toString().trim().split("\n")

const [A, B, C] = input[0].split(" ").map(x => parseInt(x))
const D = parseInt(input[1])

const now = A * 60 * 60 + B * 60 + C
const end = now + D

const H = (parseInt(end / (60 * 60))) % 24
const M = parseInt((end % (60 * 60)) / 60)
const S = end % 60
console.log(H, M, S)

File: BAEKJOON/Python/2000/2500/2530.py
"""
Title : 인공지능 시계
Link : https://www.acmicpc.net/problem/2530
"""

from sys import stdin
input = stdin.readline


if __name__ == "__main__":
    A, B, C = map(int, input().split())
    D = int(input())
    now = A * 60 * 60 + B * 60 + C
    end = now + D
    end %= 24 * 60 * 60
    H, M, S = (end // (60 * 60)) % 24, (end % (60 * 60)) // 60, end % 60
    print(H, M, S)
